plugins/backend/catalog-backend-module-okta/src/providers/OktaGroupEntityProvider.ts

```diff
@@ -112,5 +112,6 @@ export class OktaGroupEntityProvider extends OktaEntityProvider {
         locationKey: this.getProviderName(),
       })),
     });
+    this.logger.info(`Finished providing okta group resources from okta: ${this.orgUrl}`);
```

Comment:
would it make sense to show the number of groups provided?

Reply:
yup, i think so! updated all three completion logs to include counts of provided users/groups